### PR TITLE
Cdr 1695 eliminate list view

### PR DESCRIPTION
**Pull request by harringj**

### CODE REVIEW

File: access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp [hunk 1]
```diff
@@ -35,19 +35,24 @@
 
 <span class="hierarchicalTrail">  
 	<c:forEach items="${objectPath.entries}" var="pathEntry" varStatus="status">
-		<c:if test="${!param.hideLast || !status.last}">
 			<c:if test="${!status.first}">
```
Comment (bbpennel) [inline access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp]: unindent me now that the outer if is gone

File: access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp [hunk 2]
```diff
@@ -35,19 +35,24 @@
...
 			<c:choose>
-				<c:when test="${status.last && param.linkLast != true}">
+				<c:when test="${status.last}">
 					<c:out value="${pathEntry.name}" />
 				</c:when>
 				<c:otherwise>
-					<c:url var="shiftFacetUrl" scope="page" value="${queryPath}/${pathEntry.pid}${shiftFacetUrlBase}"></c:url>
+					<c:choose>
+						<c:when test="${pathEntry.resourceType == searchSettings.resourceTypeAggregate}">
+							<c:url var="shiftFacetUrl" scope="page" value="record/${pathEntry.pid}${shiftFacetUrlBase}"></c:url>
```
Comment (bbpennel) [inline access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp]: Its not doing any harm, but you can safely remove `${shiftFacetUrlBase}`.  That variable adds facet and keyword parameters from the users current search, which wouldn't do anything on a full record page

File: access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp (hunk 1 above)
Comment (harringj) [inline access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp]: Done!

File: access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp (hunk 2 above)
Comment (harringj) [inline access-common/src/main/resources/META-INF/resources/jsp/util/pathTrail.jsp]: And done!

**Pull request merged**